asset-transfer-basic/rest-api-typescript/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "asset-transfer-basic",
+  "version": "1.0.0",
+  "description": "Asset Transfer Basic REST API implemented in TypeScript",
+  "main": "dist/index.js",
+  "dependencies": {
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1",
+    "helmet": "^4.6.0",
+    "http-status-codes": "^2.1.4",
+    "pino": "^6.11.3",
+    "pino-http": "^5.5.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.12",
+    "@types/node": "^15.12.4",
+    "@types/pino": "^6.3.8",
+    "@types/pino-http": "^5.4.1",
+    "@typescript-eslint/eslint-plugin": "^4.28.0",
+    "@typescript-eslint/parser": "^4.28.0",
+    "eslint": "^7.29.0",
+    "eslint-config-airbnb-typescript": "^12.3.1",
+    "eslint-plugin-import": "^2.23.4",
+    "pino-pretty": "^5.0.2",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.3.4"
+  },
+  "scripts": {
+    "prebuild": "npm run lint",
+    "build": "tsc",
+    "clean": "rimraf ./dist",
+    "lint": "eslint . --ext .ts",
```

Comment:
Quite liked using this setup for `weft`  https://github.com/hyperledgendary/weftility/blob/6fea022ad42dadd0dd16b060448b596481295827/package.json#L12

Reply:
Don't forget weft can also be used as a module to parse different forms of identity :-)

Reply:
Aha, I've stolen that, thanks!